### ISSUE TITLE
Option to empty fields when data is unavailable from source

### ISSUE THREAD

**Issue by l-t-k** [beetbox/beets#1173]
Is it possible to scrub or remove tags from files that should be empty following the musicbrainz database, but comes with some value prior to tagging?

For example, I tagged some files which contained a (wrong) composer, but the composer field in the MB database was empty. Therefore, the tagger didn't removed the composer-tag from the files. In my view, the tagger should remove the composer-tag from the file by default if it is empty in the MB-database

**Comment by kooimens** [comment]
#934 Basically the same

**Comment by Freso** [comment]
Not quite the same as I read the two issues. 934 is about tagging the file from a completely blank slate, ie., removing *all* data from the file before writing new data to. I read this issue more like "file has cat. no. data but MusicBrainz doesn't, so clear local data". Using composers as example: with 934 composers would (currently) always be wiped, but with this, they would not be touched, as beets has no knowledge of composers from MB. There's a subtle, but IMHO important, difference between the two approaches.

**Comment by sampsyo** [comment]
Agreed: related but slightly different. They should perhaps be solved both at once.
